lib/storage_tables/attachable/model.rb

```diff
@@ -23,7 +23,7 @@ def stored_one_attachment(name, class_name:)
           end
 
           has_one :"#{name}_storage_attachment", class_name: class_name.to_s, inverse_of: :record,
-                                                 foreign_key: :record_id
+                                                 foreign_key: :record_id, dependent: :destroy
```

Comment:
Is deze nodig?

Reply:
Nope was blijven hangen, is nu weggehaald